src/aliases.jl

```diff
@@ -46,7 +45,6 @@ function number_of_vertices end
 @alias nmaxcells number_of_maximal_cells            # decided to have but not used anywhere
 @alias nmaxcones number_of_maximal_cones
 @alias nmaxpolyhedra number_of_maximal_polyhedra    # decided to have but not used anywhere
-@alias npartitions number_of_partitions             # decided to have but not used anywhere
```

Comment:
please keep these two lines in this file to provide the alias `npartitions` as documented in this [HackMD](https://hackmd.io/nRnyfrSxTVe5CzXidB1cBQ?both#number-of-%E2%80%A6)

Reply:
Ah, sorry, somehow I thought that this was the old name from Nemo.